scripts/device-catalog/support.json

```diff
@@ -2,15 +2,15 @@
   "arc": {},
   "arm": {
     "circuitdojo_feather_nrf9160": {
-      "level": "quickstart",
+      "level": "verified",
       "quickstart": "/hardware/nrf91/zephyr-quickstart"
     },
     "nrf9160dk_nrf9160": {
-      "level": "quickstart",
+      "level": "continuously-verified",
       "quickstart": "/hardware/nrf91/zephyr-quickstart"
     },
     "qemu_cortex_m3": {
-      "level": "quickstart",
+      "level": "continuously-verified",
       "quickstart": "/hardware/virtual-devices/zephyr-quickstart"
     },
     "qemu_x86": {
```

Comment:
Since the refactor doesn't support the level anymore, should we change the `qemu_x86` to be `continuously-verified` as well?

Reply:
Thanks for catching this. I merged without making this change but have fixed it with #220.